engine/src/main/java/org/terasology/rendering/dag/RenderPipelineExecutor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag;
+
+import java.util.List;
+
+/**
+ *
+ */
+public class RenderPipelineExecutor {
+
+    private List<PipelineTask> taskList;
+
+    public RenderPipelineExecutor(List<PipelineTask> taskList) {
+        this.taskList = taskList;
+    }
+
+    public void execute() {
+        taskList.forEach(PipelineTask::execute);
+    }
+}
```

Comment:
@emanuele3d maybe debugging functionality can be placed here. Do you think it is ok to have this one?